cfn/paws-collector.template

```diff
@@ -529,9 +535,15 @@
                   "paws_api_client_id":{
                       "Ref":"PawsClientId"
                   },
+                  "paws_kms_key_arn":{
+                      "Ref":"LambdaKmsKey"
```

Comment:
should we use `Fn::GetAttr` here?
```
"Fn::GetAtt": [
    "LambdaKmsKey",
    "Arn"
  ]
```

Reply:
yep thats what we need. i misread my cfn cheatsheet, i thought that returned he arn